crates/bevy_render/src/extract_param.rs

```diff
@@ -107,7 +98,7 @@ where
     }
 }
 
-impl<'w, 's, P: SystemParam> DerefMut for Extract<'w, 's, P>
+impl<'w, 's, P> DerefMut for Extract<'w, 's, P>
```

Comment:
The bounds were weakened here, but IMO correctly. ReadOnlySystemParam implies SystemParam.